frame/nomination-pools/src/lib.rs

```diff
@@ -2160,6 +2160,30 @@ impl<T: Config> Pallet<T> {
 		None
 	}
 
+	/// Returns the points to balance conversion for a specified pool.
+	///
+	/// If the pool ID does not exist, it returns an error. Used by runtime API.
+	pub fn api_points_to_balance(pool_id: PoolId) -> BalanceOf<T> {
+		if let Some(pool) = BondedPool::<T>::get(pool_id) {
+			pool.points_to_balance(pool.points)
```

Comment:
how is this exactly being used? this function should get `pool_id` and some `point` and convert it. `pool.points_to_balance(pool.points)` simply returns the pool's stake 🤷‍♂️